SYCL/Plugin/level_zero_sub_sub_device.cpp

```diff
@@ -121,15 +97,15 @@ int main(void) {
 
   // watch out device here
   auto subdevices =
-      devices[1]
+      devices[0]
```

Comment:
I'd probably change line 95 to

```
  device d;
```

and then just use `d` here, with lines 96-98 removed.